drivers/sx126x/sx126x.c

```diff
@@ -199,57 +203,69 @@ int sx126x_init(sx126x_t *dev)
         DEBUG("[sx126x] init: cmd status %d\n", radio_status.cmd_status);
     }
 
+    /* Radio Rx timeout timer stopped on preamble detection */
+    sx126x_stop_timer_on_preamble(dev, true);
+
     return res;
 }
 
 uint32_t sx126x_get_channel(const sx126x_t *dev)
 {
+    DEBUG("[sx126x.c] sx126x_get_radio_status \n");
```

Comment:
```suggestion
    DEBUG("[sx126x]: sx126x_get_radio_status \n");
```

Nitpick, but its fits other logs style better IMO